shipped-101-first-project/src/posts/files/shipped-101-first-project/2.md

```diff
@@ -15,16 +15,12 @@ If you are using Microsoft Windows you will want to open a Command Prompt instea
 
 Near the end of the previous example we clicked "Build Project". If it is succesful we will be provided with a popup message which is a curl command to pull our project down to our local environment from Shipped. 
 
-The curl command you are to copy from the Shipped UI will look very much like this:
-
-<pre>'"curl 'http://shipped-api.shipped-cisco.com/cli/b56b264b-fa6d-11e4-953e-0242ac11000a/bootstrap?api_token=qIexsFzXzrlcztxcdJJFuyZHRCovgpVv' | bash"</pre></li>
+The curl command you are to copy from the Shipped UI will download the latest shipped cli and second command will perform the local bootstrap.
```

Comment:
Please change to:

The curl command you ~~are to~~ copy from the Shipped UI ~~will~~ download***s*** the latest shipped ***CLI*** and ***the*** second command ~~will~~ perform***s*** the local bootstrap